### PR TITLE
Add NVS usage example using the native Rust API

### PR DESCRIPTION
**Pull request by punkto**
Please note that I have added a new dependence (postcard) just for the example. I don't know if that is OK for you or it is better to use something already added (json?).

### CODE REVIEW

**Comment by ivmarkov** [comment]
Dependencies which are only relevant for the `[examples]` section should be added in the `[dev-dependencies]` section.

File: examples/nvs_get_set_raw_storage.rs [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    {
+        let tag_raw_str_data: &mut [u8] = &mut [0; 100];
+
+        match nvs.get_raw(tag_raw_str, tag_raw_str_data) {
+            Ok(v) => {
+                if let Some(the_str) = v {
+                    info!("{:?} = {:?}", tag_raw_str, from_bytes::<&str>(the_str));
+                }
+            }
+            Err(e) => info!("Couldn't get tag {} because{:?}", tag_raw_str, e),
```
Comment (ivmarkov) [inline examples/nvs_get_set_raw_storage.rs:89]: Nit: missing space

File: examples/nvs_get_set_raw_storage.rs [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+    let test_namespace = "test_ns";
+    let mut nvs = match EspNvs::new(nvs_default_partition, test_namespace, true) {
+        Ok(nvs) => {
+            info!("Got namespace {:?} from default partition", test_namespace);
+            nvs
+        }
+        Err(e) => panic!("Could't get namespace {:?}", e),
+    };
+
+    let tag_raw_u8 = "test_raw_u8";
```
Comment (ivmarkov) [inline examples/nvs_get_set_raw_storage.rs:43]: Rather than `tag`s, can we name these things `key`s?

File: examples/nvs_get_set_raw_storage.rs [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+            Ok(v) => {
+                if let Some(the_struct) = v {
+                    info!(
+                        "{:?} = {:?}",
+                        tag_raw_str,
+                        from_bytes::<StructToBeStored>(the_struct)
+                    )
+                }
+            }
+            Err(e) => info!("Couldn't get tag {} because{:?}", tag_raw_str, e),
```
Comment (ivmarkov) [inline examples/nvs_get_set_raw_storage.rs:123]: Ditto - missing space?

**Review by ivmarkov — CHANGES_REQUESTED** [review]
Really just two small nits.

**Comment by punkto** [comment]
done!

**Pull request merged**